packages/babel-helper-create-class-features-plugin/src/decorators.ts

```diff
@@ -268,6 +268,156 @@ function extractProxyAccessorsFor(
   ];
 }
 
+/**
+ * Prepend expressions to the field initializer. If the initializer is not defined,
+ * this function will add a trailing `void 0` expression
+ *
+ * @param {t.Expression[]} expressions
+ * @param {(NodePath<
+ *     t.ClassProperty | t.ClassPrivateProperty | t.ClassAccessorProperty
+ *   >)} fieldPath
+ */
+function prependExpressionsToFieldInitializer(
+  expressions: t.Expression[],
+  fieldPath: NodePath<
+    t.ClassProperty | t.ClassPrivateProperty | t.ClassAccessorProperty
+  >,
+) {
+  const initializer = fieldPath.get("value");
+  if (initializer.node) {
+    expressions.push(initializer.node);
+  } else {
+    expressions.push(t.unaryExpression("void", t.numericLiteral(0)));
```

Comment:
It's ok to do it in a follow-up if you want to keep the fixtures diff minimal, but we can make this shorter by doing

```js
  x = expr1, void expr2
```
instead of
```js
  x = expr1, expr2, void 0
```

Reply:
Good idea, I can make the change like you said. This branch is currently unreachable because the `protoInit` call is currently injected in the first _decorated_ field, which will include at least a `init_field` call in its initializer. It is probably an oversight and I will open a new PR for that.